examples/get-started/pure-js/widgets/app.js

```diff
@@ -67,5 +67,5 @@ new Deck({
       getWidth: 1
     })
   ],
-  widgets: [new FullscreenWidget({style: widgetTheme})]
+  widgets: [new ZoomWidget({style: widgetTheme}), new FullscreenWidget({style: widgetTheme})]
```

Comment:
Q: What happens if we provide these multiple times? Every render will create a new set of DOM elements?

Reply:
Yes, and they stack vertically by default (`display: block;`).

Reply:
Yes, looks like this:

<img src=https://github.com/visgl/deck.gl/assets/351828/63977c70-c395-42e9-95c7-840ba2a0c354 width=300>